### PR TITLE
input form element

### PR DESCRIPTION
**Pull request by ritik-patel05**
https://user-images.githubusercontent.com/67374176/151715667-42ebd193-5e36-4717-b91c-0b0888b4d658.mp4

### CODE REVIEW

File: css/form/input.css [hunk 1]
```diff
@@ -0,0 +1,155 @@
...
+.is-success.input:focus {
+    border-color: var(--success-hover);
+}
+
+.is-danger.input:active,
+.is-danger.input:focus {
+    border-color: var(--danger-hover);
+}
+
+/* TODO:
```
Comment (Hamdrive) [inline css/form/input.css:152]: If the feature is implemented then remove this comment

**Review by Hamdrive — COMMENTED** [review]
Clean code, looks good!

**Comment by saadahmadhere** [comment]
LGTM

**Review by saadahmadhere — COMMENTED** [review]
Nice!

**Pull request merged**